data/strings/strings.txt

```diff
@@ -3937,6 +4023,7 @@
     ja = RV用
     ko = RV용
     lt = Rekreacinio transporto aikštelės
+    lv = Treileru laukumi
```

Comment:
This translation is incorrect, IMHO.
I would say "Labierīcības automājām" or "Labiekārtojums automājām"?

Reply:
I think the best version would be "Atpūtas transportlīdzekļu labierīcības".

RVs consists of motorhomes and trailers. Yes, "treileru laukumi" is a mistake as it does not encompass motorhomes.
For motorhomes there are no approved terms, I can find both "motormāja" and "automāja" used. However, is does not encompass whole RVs either.
I can't find anything for RV in dictionaries, but "atpūtas transportlīdzeklis" is a term used in EU documentation among "atpūtas un izklaides transportlīdzklis", "transportlīdzeklis izklaidei" etc.

Reply:
Manā tehniskajā pasē ir rakstīts "Vieglais dzīvojamais" (transportlīdzeklis).

So what do you think about this?
"Dzīvojamo auto labierīcības"

Reply:
Paldies par informāciju! Man nav bijusi tuvāka pazīšanās ar treileriem un motormājām.

„Dzīvojamo auto labierīcības“ šķiet adekvāti un varbūt tad arī paliekam pie tā, jo izklausās, ka tāda man 100% skaidra varianta nav.